pkg/operator/controller/nodeobservability/controller.go

```diff
@@ -278,6 +286,10 @@ func (r *NodeObservabilityReconciler) ensureNodeObservabilityDeleted(ctx context
 	if err := r.deleteSecurityContextConstraints(nodeObs); err != nil {
 		errs = append(errs, fmt.Errorf("failed to delete SCC : %w", err))
 	}
+	if err := r.deleteNOMC(ctx, nodeObs); err != nil {
+		// TODO: blockOwnerDeletion: true, update status
+		errs = append(errs, fmt.Errorf("failed to delete SCC : %w", err))
```

Comment:
failed to delete MCO, right?

Reply:
Good catch. Changed it to `nodeobservabilitymachineconfig`